boards/x86/intel_adl/Kconfig.defconfig

```diff
@@ -33,7 +33,7 @@ config ACPI
 	default y
 
 if ACPI
-config HEAP_MEM_POOL_SIZE
+config HEAP_MEM_POOL_ADD_SIZE_ACPI
```

Comment:
if various subsystems starts to use `ADD_SIZE` then the default for this board should probably be recalculated as the current default is probably chosen based on the subsystems / libs default enabled for this board.

But it would be the board maintainers who would actually know how to determine the optimal value for the `ADD_SIZE`.

This comment applies for boards in general where `HEAP_MEM_POOL_SIZE` is now changed to an `ADD_SIZE`.

I'm fine with the current approach of changing to `ADD_SIZE`, and then leave any optimization (heap size reduction) as followup work for board maintainers.